prometheus_flask_exporter/__init__.py

```diff
@@ -263,12 +263,18 @@ def register_endpoint(self, path, app=None):
             (by default it is the application registered with this class)
         """
 
-        if is_running_from_reloader() and not os.environ.get('DEBUG_METRICS'):
-            return
```

Comment:
Sorry, I should have been clearer: I think we should log the warning here once, at startup, and just return then as before, so we don't even attempt registering the metrics endpoint that will be unavailable under the reloader anyway.